ANSWERS.md

```diff
@@ -1,4 +1,19 @@
-- [ ] Why would you use class component over function components (removing hooks from the question)?
-- [ ] Name three lifecycle methods and their purposes.
-- [ ] What is the purpose of a custom hook?
-- [ ] Why is it important to test our apps?
\ No newline at end of file
+- [x] **Why would you use class component over function components (removing hooks from the question)?**
+
+-- Functional components don't have state, life cycle methods, or _this_, whereas, class components do.
+
+- [x] **Name three lifecycle methods and their purposes.**
+
+-- *constructor()* - Typically, in React constructors are only used for two purposes: Initializing local state by assigning an object to this.state and binding event handler methods to an instance.
```

Comment:
Technically `constructor` isn't a lifecycle method. It's part of JavaScript classes. But you're right on the way the constructor is used with React class components. 👍